torch/_dynamo/variables/builder.py

```diff
@@ -584,15 +584,15 @@ def wrap_sym(self, value: Union[torch.SymInt, torch.SymFloat]):
         elif is_constant_source(self.get_source()):
             return self.tx.output.register_attr_or_module(
                 value,
-                re.sub(r"[^a-zA-Z0-9]+", "_", self.name),
+                normalize_attr_name(self.name),
```

Comment:
Sadly, we need a little extra normalization here, so we can have a consistent lookup.

Reply:
This doesn't make sense to me. You'll have to explain more. Alt, in a pinch, I can check out your PR and take a look at the error.

Reply:
Yes, will add comments here to explain. Do not worry about pulling it down, its not ready yet, and not a good use of your time!